rubyforgood/circulate
# Initial dev data

# What it does

This PR starts to setup a better way for developers to have realistic data to work with.

# Why it is important

We want it to be as easy as possible for new developers to get started working on the project.

# Implementation notes

* This first round includes items, categories, borrow policies, and documents. Members and loans will be part of a future round.

# How to Verify

Pull down the branch and run `bin/setup`, and `bin/reset`. Both should run without error and result in a few hundred items and their categories, borrow policies being added to the database. There should also be documents added (which can be viewed in the admin under Settings -> Documents).
